src/frontend/src/pages/PPQPage/messages.ts

```diff
@@ -0,0 +1,36 @@
+export default {
```

Comment:
Just wondering. To create these message are we following some standards? Or it is always custom?

Reply:
@Bottle7 @kushal-arora-fw  any idea?

Reply:
I have some ideas. we can discuss later. For now it is fine.

Reply:
Not really a global standard per say. But this is something I have followed in one of my previous projects and has worked out well.

The idea was to remove the usage of string in the jsx files/components, and have standard messaging framework. And we be ready for language switches when we bring localization/internationalization for French into the system.

Ofcourse, I have some enhancements in mind. However, as that can be for future.